DEVELOPERS.md

```diff
@@ -68,6 +68,18 @@
          `Console` tab to view printed logs and errors, the `Element` tab to
          inspect the HTML and CSS, the `Network` tab for network calls, etc).
 
+#### Debugging Add-in: Alternate way
+If the above method to debug the add-in is not working, one can try the following steps:
+   1. Delete the `C:\Users{username}.office-addin-dev-certs` folder.
```

Comment:
You may change this to `C:\Users\{username}\.office-addin-dev-certs`

Reply:
@Aniruddha-Shriwant 
Thanks for the typo.